webapp/app/actions/authActions.test.js

```diff
@@ -1,16 +1,17 @@
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import * as authActions from './authActions';
-import * as types from '../constants/actionTypes';
+import actionTypes from '../constants/actionTypes';
 
+const { AUTH } = actionTypes;
```

Comment:
QUESTION: do we need this intermediate variable here? 

PREF: destructure it in the import instead ( if you are looking for clarity, maybe assign it a new name :authActionTypes ?)

`import { AUTH: authActionTypes } from '../constants/actionTypes';
...
type: authActionTypes.LOGIN_REQUEST
`

Reply:
`Import` can't destructure objects, are Just Named Imports, you need to import the object and then destructure it.
By the way Syntax is wrong it shall be
```
import { NamedImport as Alias } from 'foo'
```

Reply:
I started adding that reply to your changes to do it like this

const { AUTH: authAuctionTypes } syntax is correct there, then i switched to the import and left that one there, thanks